keps/0029-operator-dependencies.md

```diff
@@ -102,7 +102,7 @@ The `zookeeper-operator` task specification is equivalent to `kudo install zooke
 ```yaml
 tasks:
 - name: demo
-  kind: Operator
+  kind: KudoOperator
```

Comment:
I ended up renaming `s/Operator/KudoOperator` thinking about _other operators_ we could deploy in the future after we have a mechanism in place. But this is up for discussion.